logic/src/commonJvmAndroid/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -188,7 +192,7 @@ class CallManagerImpl internal constructor(
         message: Message.Signaling,
         content: MessageContent.Calling,
     ) = withCalling {
-        callingLogger.i("$TAG - onCallingMessageReceived called")
+        callingLogger.i("$TAG: { \"message\" : ${message.toLogString()}}")
```

Comment:
Maybe keep the same information as before to know where this message log came from by looking at the logs 😄 

```suggestion
        callingLogger.i("$TAG - onCallingMessageReceived called: { \"message\" : ${message.toLogString()}}")
```

Reply:
We will the message twice now, once in `ApplicationMessageHandler` and again here.  I don't think we need to log it here.

Reply:
is it logged on ApplicationMessageHandler tho? doesn't hat just to Message.Regular?

Reply:
Right, then all is good